package/MDAnalysis/coordinates/GRO.pyx

```diff
@@ -116,6 +116,10 @@
 from ..core import flags
 from ..exceptions import NoDataError
 from ..lib import util
+cimport cython
+@cython.boundscheck(False)  # Deactivate bounds checking
```

Comment:
these decorators aren't doing what you are expecting here, they won't apply to the whole file.  I think they're probably just modifying the Timestep object.

You probably need to decorate the class method directly